packages/main/src/StepInput.ts

```diff
@@ -379,6 +390,14 @@ class StepInput extends UI5Element implements IFormInputElement {
 		}, 0);
 	}
 
+	_onInput(e: InputEvent) {
```

Comment:
`InputEvent` is wrong here - this is the event type that will be fired by the native input (so in Input.ts it's normal that the event listener of this type) - here you need the correct `InputEventDeatil` from the `Input.ts` event types.